state/grants.go

```diff
@@ -646,6 +646,148 @@ func GetHardCodedGrants() []HardGrant {
+			// [Factom Inc.-17] Anchor Master -- 660 FCT
+
+			// Factom Inc. total: 660 FCT
+			HardGrant{207325, 660e8, validateAddress("FA3jySUFtLXb1VdAJJ5NRVNYEtZ4EBSkDB7yn6LuKGQ4P1ntARhx")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// [The 42nd Factoid AS-Centis BV-Factom Inc.—DBGrow Inc-TRgg3R LLC-6] Guide Compensation -- 9000 FCT
+			// Guide Compensation 2019-07-21 - 2019-08-17
+
+			// Factom Inc. total: 1800 FCT
+			HardGrant{207325, 1800e8, validateAddress("FA2teRURMYTdYAA97zdh7rZDkxNtR1nhjryo34aaskjYqsqRSwZq")},
+
+			// TRGG3R LLC total: 1800 FCT
+			HardGrant{207325, 1800e8, validateAddress("FA2oecgJW3XWnXzHhQQoULmMeKC97uAgHcPd4kEowTb3csVkbDc9")},
+
+			// DBGrow Inc total: 1800 FCT
+			HardGrant{207325, 1800e8, validateAddress("FA3HSuFo9Soa5ZnG82JHqyKiRi4Pw17LxPTo9AsCaFNLCGkXkgsu")},
+
+			// The 42nd Factoid AS total: 1800 FCT
+			HardGrant{207325, 1800e8, validateAddress("FA3AEL2H9XZy3n199USs2poCEJBkK1Egy6JXhLehfLJjUYMKh1zS")},
+
+			// Centis BV total: 1800 FCT
+			HardGrant{207325, 1800e8, validateAddress("FA2hvRaci9Kks9cLNkEUFcxzUJuUFaaAE1eWYLqa2qk1k9pVFVBp")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// [Factomize-7] Core and General Development -- 8747 FCT
+
+			// Factomize total: 8747 FCT
+			HardGrant{207325, 8747e8, validateAddress("FA3nsSjUy5uSkqMEug8t3VcehZn5w2ciSMpgqFEEsMRwMrHoa9k3")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// [Sphereon-7] Core development continuation -- 8093 FCT
+
+			// Sphereon total: 8093 FCT
+			HardGrant{207325, 8093e8, validateAddress("FA3P6cLv2Ct2P6SStqhhqagFp9u8kpca9XyJHbhB68v1jeZNFpG8")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// [Bedrock Solutions-De Facto-Crypto Logic-The Factoid Authority-4] Factom Open Node Continuity -- 1126 FCT
+
+			// The Factoid Authority total: 188 FCT
+			HardGrant{207325, 188e8, validateAddress("FA2LV4s7LKA9BTgWaJNvcr9Yq8rpiH2XD3vEPY3nwSiNSrnRgkpK")},
+
+			// Crypto Logic total: 188 FCT
+			HardGrant{207325, 188e8, validateAddress("FA29wMUjN38BVLbJs6dR6gHHdBys2mpo3wy565JCjquUQTGqNZfb")},
+
+			// Bedrock Solutions total: 375 FCT
+			HardGrant{207325, 375e8, validateAddress("FA2FqYZPfBeRWq7fWSFEhassT5zpMQZm8jwus3yWbzeN3PZPWybm")},
+
+			// De Facto total: 375 FCT
+			HardGrant{207325, 375e8, validateAddress("FA2YeMbN8Z1SsT7Yqw6Np85kWwtFVg2CyJKMDFnuXTawWuWPtzvX")},
+			// --------------------------------------------------------
+
+			// --------------------------------------------------------
+			// [Factom Inc.-18] Protocol Development -- 38640 FCT
+
+			// Note: 300 FCT to be paid for a third sponsor has been removed from this grant due to
```

Comment:
The 3rd Factom, Inc sponsor address has been posted here.  The sponsor agreed to take over just a few hours after the grant QA period closed:  https://factomize.com/forums/threads/factom-inc-015-core-development.2052/#post-18840

I will add back in the payout address with David Kuiper's FCT address for the sponsor address as mentioned here.  The sponsor address is FA2FqYZPfBeRWq7fWSFEhassT5zpMQZm8jwus3yWbzeN3PZPWybm